### DISCUSSION TITLE
Copilot SOC2/Other security documentation

### DISCUSSION THREAD

**Discussion by mackthehack** [community/community#65221] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Working in a regulated industry requires me to do due diligence for partners of our service. We are looking optimally a SOC2 or any other audit documentation specific to the Copilot product. Is there any available prior to a purchase?

**Comment by davecheney** [comment, marked as answer]
Those documents are available to CfB customers. Please work with your seller for details.

**Reply by mackthehack** [reply]
What magic does a person need to perform to "work with your seller"? I am unable to get a sales rep to talk to me.
